support-frontend/assets/components/paypalExpressButton/PayPalExpressButton.jsx

```diff
@@ -46,7 +42,7 @@ type PropTypes = {|
 // 1. Loading this iframe is an expensive operation which causes an obvious visual re-render
 // 2. We don't want to have to re-bind handlers which interact with the iframe
 //    (e.g. the handler bound in getPayPalOptions)
-export class PayPalRecurringButton extends React.Component<PropTypes> {
+export class PayPalExpressButton extends React.Component<PropTypes> {
```

Comment:
I've renamed the component to `PayPalExpressButton` to reflect the fact that it is no longer used only by recurring contributions. PayPal Express is the actual PayPal api that we are using.